superset/charts/schemas.py

```diff
@@ -816,6 +821,13 @@ class ChartDataQueryObjectSchema(Schema):
         "as `having_druid`.",
         deprecated=True,
     )
+    druid_time_origin = fields.String(
+        description="Starting point for time grain counting on legacy Druid "
+        "datasources. Used to change e.g. Monday/Sunday first-day-of-week. "
+        "This field is deprecated and should be passed to `extras` "
+        "as `druid_time_origin`.",
+        allow_none=True,
+    )
```

Comment:
Turns out this was missing from the chart data schema. Also added to deprecated field mapping in `query_object.py`.